.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Request for new functionality
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```

Comment:
Maybe not "happen" but "change". I think what we want is a description of the feature here, right? Maybe we should just say this ;)

Reply:
It's funny that these are the default github issues templates 😄 Will adjust it though